mapie/tests/test_regression.py

```diff
@@ -39,6 +40,19 @@
 
 random_state = 1
 
+gb = GradientBoostingRegressor(
+            random_state=random_state
+            )
```

Comment:
```suggestion
gb = GradientBoostingRegressor(random_state=random_state)
```

Reply:
I also suggest moving it directly into the test (before instantiating the `mapie` estimator) to better understand where `gb` is being used.